dspace-api/src/test/java/org/dspace/uclouvain/itemEnhancer/UCLouvainItemEnhancerTest.java

```diff
@@ -600,4 +605,212 @@ public void testConsumerMultipleConfigurationsAndMultipleEnhancement() throws Ex
             return ite.getTargetItem().getID().toString().equals(publication2UUID);
         }));
     }
+
+    /**
+     * Simple test for the item enhancer poller with only one relation being affected.
+     * Here we link a publication and a person with the person name:
+     * - person (dc.title) <-> publication (dc.contributor.author)
+     * 
+     * When we modify the dc.title of the person, it must create a new entry in the database.
+     * Then, we trigger the poller to read the database and process the updates.
+     * At the end, the dc.contributor.author of the publication should be the same as dc.title of the person.
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void pollerTestWithOnePublicationAuthorRelation() throws Exception {
+        // Create a simple author-publication relation.
+        // The publication is linked to the author via the name.
+        context.turnOffAuthorisationSystem();
+        Item person = ItemBuilder.createItem(context, collection)
+            .withEntityType("Person")
+            .withTitle("Pierre Kiroul")
+            .build();
+        UUID personAuthority = person.getID();
+
+        Item publication = ItemBuilder.createItem(context, collection)
+            .withTitle("Test publication0")
+            .withEntityType("Publication")
+            .withAuthor("Pierre Kiroul", personAuthority.toString())
+            .build();
+        context.restoreAuthSystemState();
+
+        context.commit();
+        person = context.reloadEntity(person);
+        publication = context.reloadEntity(publication);
+
+        // Modify the dc.title of the person
+        context.turnOffAuthorisationSystem();
+        itemService.setMetadataSingleValue(context, person, "dc", "title", null, null, "Jean Gloutitou");
+        itemService.update(context, person);
+        context.restoreAuthSystemState();
+
+        context.commit();
+        person = context.reloadEntity(person);
+        publication = context.reloadEntity(publication);
+
+        // We now have one entry to process in the database.
+        List<ItemToEnhance> itemToEnhanceEntries = uclouvainItemEnhancerService.retrieveAllItemsToUpdate(context);
```

Comment:
here you could create a `uclouvainItemEnhancerService.countEntries()` method when the limit param has been created